.github/workflows/release.yaml

```
@@ -6,18 +6,34 @@
 name: Publish Python Package to PyPI
 on:
   push:
-    tags:
-      - '*.*.*'
+    branches:
+      - main
```

```suggestion
      - main
permissions:
  contents: write
```

Set the correct permissions to avoid the PAT.